streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedSegmentedBytesStore.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
+
+public class RocksDBTimestampedSegmentedBytesStore {
```

Comment:
Add this as dummy because I reference it in JavaDocs of `AbstractSegments` -- will be implemented in follow up PR.

Reply:
Not a big deal, but I'm not a huge fan of dead code (here or below). IMHO, it would have been better to avoid adding the reference in the javadoc and thereby avoid the obligation to add both of these un-implemented, un-tested, and un-used classes.

No change requested, just sharing my thoughts ;) I know you're queuing up the implementations in another PR right away.

Reply:
While I agree, I did it this way, because otherwise, updating the Javadoc would have most likely slipped...